src/ListView/__mocks__/MockListManager.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import update from 'immutability-helper'
+import { mockListItems } from './mockListItems'
+import ListView from '../ListView'
+import ListViewItem from '../ListViewItem'
+import ListViewItemContainer from '../ListViewItemContainer'
+import ListViewItemExpansion from '../ListViewItemExpansion'
+
+class MockListManager extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      items: mockListItems
+    }
+    this.itemClicked = (e, i) => {
+      this.setState(prevState =>
+        update(prevState, {
+          items: {
+            [i]: {
+              active: {
+                $set: !prevState.items[i].active
+              }
+            }
+          }
+        })
+      )
+    }
+    this.childClicked = (e, i, j) => {
+      this.setState(prevState =>
+        update(prevState, {
+          items: {
+            [i]: {
+              children: {
+                [j]: {
+                  active: {
+                    $set: !prevState.items[i].children[j].active
+                  }
+                }
+              }
+            }
+          }
+        })
+      )
+    }
+  }
+  render() {
+    return (
+      <ListView>
+        {this.state.items.map((item, i) => (
+          <ListViewItem isActive={item.active} key={i}>
```

Comment:
Using index as key has some un-desirable sideffects, see here:
https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318
i think it might be better to assign a uuid to these items...